cl-gopher.lisp

```diff
@@ -478,16 +478,23 @@
             (if (and (>= (length uri) 9) (equal "gopher://" (subseq uri 0 9)))
                 (quri:uri uri)
                 (quri:uri (format nil "gopher://~a" uri))))
-           (path (quri:uri-path uri))
+           (path (quri:url-decode (quri:uri-path uri)))
            (item-type (compute-item-type uri path))
            (selector (compute-selector uri path))
+           (tab-split-selector (uiop:split-string selector :separator '(#\Tab)))
```

Comment:
Can we use the standard `split-sequence` rather than relying on `uiop`?

Reply:
But wait, there's no `split-sequence` in CLHS 0_o

UIOP is a de-facto standard and all the implementations package it. I see no problem with using it there :) I mean, I can rewrite it using `subseq` and friends, but why not use something that makes it better and is provided by all the modern implementations?

Reply:
Sorry, by "standard" I meant "what I'm already using in `cl-gopher`".

I have a `split-sequence` dependency: https://github.com/knusbaum/cl-gopher/blob/master/cl-gopher.asd#L6
Which I use in `read-gopher-line`: https://github.com/knusbaum/cl-gopher/blob/master/cl-gopher.lisp#L270

I would prefer to either use that `split-sequence` or replace its use with `uiop:split-string` and remove the dependency to be consistent.